### DISCUSSION TITLE
How do I get Output?

### DISCUSSION THREAD

**Discussion by randy-johnson** [czproject/git-php#68] [Q&A] [answered]
Do I have to do anything special to see the output of the commands I run.  I don't see any output unless there was a fatal exception.  Once I got everything worked any output stopped.  The only way I know something worked is by checking the files.

Here is my code:

```

<?php
require __DIR__ . '/vendor/autoload.php';
// create repo object
$repo = new Cz\Git\GitRepository('/home/test_repo/public_html');

$repo->pull('origin');

$repo->checkout('first-draft');

```

**Comment by janpecha** [comment, marked as answer]
Hello, you can use `execute()`:

```php
$output = $repo->execute('pull origin');
```

Currently there is no other way.
